src/rpc_client/src/meta_client.rs

```diff
@@ -477,23 +478,69 @@ impl HummockMetaClient for MetaClient {
             .unwrap())
     }
 
-    async fn get_version_deltas(
+    async fn reset_current_version(&self) -> Result<HummockVersion> {
```

Comment:
Alternatively we can define these methods inner `Impl MetaClient`, so that no need to extend HummockMetaClient trait. It's necessary to define a method in HummockMetaClient only when we need use it in MockHummockMetaClient.